include/range/v3/algorithm/contains.hpp

```diff
@@ -0,0 +1,59 @@
+/// \file
+// Range v3 library
+//
+//  Copyright Johel Guerrero 2019
+//
+//  Use, modification and distribution is subject to the
+//  Boost Software License, Version 0.0. (See accompanying
+//  file LICENSE_0_0.txt or copy at
+//  http://www.boost.org/LICENSE_0_0.txt)
```

Comment:
Hmm. I copy pasted this from the implementation of `view::trim` which already had my name, which I copy pasted from some other header at the time. What should I do, @ericniebler?

Reply:
We have four files checked in with BSL version 0: `algorithm/ends_with.hpp` and its test file have your copyright, and `algorithm/equal.hpp` and `algorithm/find_first_of.hpp` have @ericniebler's copyright. `https://www.boost.org/LICENSE_0_0.txt` returns a 404, which suggests to me there is no such license?

Reply:
Actually, I got it from `ends_with.hpp`. Since it uses `equal.hpp`, I must have gotten it from there. A quick `egrep '/LICENSE_0_0' *` at `algorithm` reveals that `find_first_of.hpp` also suffers from this.